Lib/test/test_exceptions.py

```diff
@@ -188,7 +188,7 @@ def check(src, lineno, offset, encoding='utf-8'):
                 if not isinstance(src, str):
                     src = src.decode(encoding, 'replace')
                 line = src.split('\n')[lineno-1]
-                self.assertEqual(cm.exception.text.rstrip('\n'), line)
+                self.assertIn(line, cm.exception.text.rstrip('\n'))
```

Comment:
You don't really need the `rstrip()` any more.
```suggestion
                self.assertIn(line, cm.exception.text)
```